rest/route/project.go

```diff
@@ -401,6 +413,25 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 	return responder
 }
 
+func (h *projectIDPatchHandler) hasCommitQueuePatchDefinition(pRef *model.APIProjectRef) (bool, error) {
+	for _, alias := range pRef.Aliases {
+		if model.FromAPIString(alias.Alias) == evergreen.CommitQueueAlias {
+			return true, nil
+		}
+	}
+
+	aliases, err := h.sc.FindProjectAliases(model.FromAPIString(pRef.Identifier))
```

Comment:
the same here: if all the definitions were deleted in this request, I think it won't be written to the database until 
https://github.com/evergreen-ci/evergreen/blob/d5aed3f8061abe38fc93f53b836f1693eccbb651/rest/route/project.go#L364

Reply:
Good catch, thanks!